src/modules/clark/metric-module/metrics.routes.ts

```diff
@@ -14,4 +14,8 @@ export const METRICS_ROUTES: ProxyRoute[] = [
         method: HTTPMethod.GET,
         path: "/users/metrics",
     },
+    {
+        method: HTTPMethod.GET,
+        path: "/learning-objects/stats",
+    },
```

Comment:
when do we need to have `target: envConfig.getUri(CLARK_SERVICE_URI)` in our routes? Or is it always included for any service-refactors?

Reply:
+1 also curious as to why no target

Reply:
You do need the target here